tests/statement_serializer_tests/schema/index.cpp

```diff
@@ -52,3 +52,22 @@ TEST_CASE("statement_serializer index") {
     }
     REQUIRE(value == expected);
 }
+
+TEST_CASE("trying schema_status index") {
+    struct User {
+        int id = 0;
+        std::string name;
+    };
+    auto storage = make_storage("",
+                                make_index("user_name", &User::name),
+                                make_table("users",
+                                           make_column("id", &User::id, primary_key(), autoincrement()),
+                                           make_column("name", &User::name)));
+    storage.sync_schema(true);
+    storage.sync_schema_simulate(true);  // this will call in storage_t:
```

Comment:
what this test tests?

Reply:
I did not write this test either!!

Reply:
it exists in your diff. It means you commit it

Reply:
The diff doesn't lie, obviously.